docs/interactions.rst

```diff
@@ -0,0 +1,131 @@
+:orphan:
+
+.. currentmodule:: nextcord
+
+.. _interactions:
+   
+Slash Commands
+==============
+
+Since Discord has added interaction commands a feature with many possibilities, we at Nextcord have decided to add them to our fleet of features.
+
+This doc will explain the innerworkings and how to use interaction commmands.
+
+We suggest you learn how to make regular commands before looking through here, we suggest looking at the :doc:`quickstart`
+
+How To Make A Simple Slash Command
+----------------------------------
+
+This right here is a simple ping pong command made with Nextcords slash feature.
+
+.. code-block:: python3
+
+    @bot.slash_command(name="ping")
+    async def ping(interaction):
+        await interaction.response.send_message("Pong!")
+        
+The way it works is that you use the :meth:`~Client.slash_command` function to interact with the Discord API. The ``name`` parameter is the name of your slash command.
```

Comment:
> function to interact with the Discord API.
This is awfully worded imo, also unnecessary as the entire lib is about interacting with the discord api?